api/tests/integration/infrastructure/repositories/user-repository_test.js

```diff
@@ -990,6 +990,26 @@ describe('Integration | Infrastructure | Repository | UserRepository', () => {
 
   });
 
+  describe('#updateLastPixTermsOfServiceAccepted', () => {
+    let userId;
+
+    beforeEach(() => {
+      userId = databaseBuilder.factory.buildUser({ mustValidateTermsOfService: true , lastTermsOfServiceValidatedAt: null }).id;
+      return databaseBuilder.commit();
+    });
+
+    it('should validate the last terms of service and save the date of acceptance ', async () => {
+      // when
+      const actualUser = await userRepository.updateLastPixTermsOfServiceAccepted(userId);
+
+      // then
+      expect(actualUser.lastTermsOfServiceValidatedAt).to.be.exist;
```

Comment:
Vérifier que c'est bien une date

Reply:
Je propose de vérifier:
- que ce soit une date (null/undefined): `expect(oneAM).to.be.a("Date");` 
- que la date soit passée avec [below(now)](https://github.com/GradedJestRisk/js-training/blob/master/test/chai/test.js#L115)

Reply:
J'ai rajouté le check du type to.be.a('Date'). Thanks pour le pointeur perso :)